src/components/RecipeCard.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { TimeIcon } from '@chakra-ui/icons';
+import Img from 'gatsby-image';
+import { graphql } from 'gatsby';
+
+interface Props {
+  data: GatsbyTypes.RecipeCardFragment;
+}
+
+function RecipeCard({ data }: Props): JSX.Element {
+  return (
+    <div style={{padding: '50px 10px 50px 10px', marginRight: '25px'}}>
+      <div style={{ marginBottom: '5%'}}>
+        <h3
+          style={{
+            position: 'absolute',
+            fontWeight: 850,
+            textAlign: 'left',
+            fontSize: 14,
+          }}
+        >
+          {data.title}
+        </h3>
+        <TimeIcon
+          style={{ marginTop: 3, color: '#65BF73', position: 'absolute', marginLeft: '78%'}}
+        />
+        <h1 style={{ fontSize: 14, fontWeight: 'bold', color: '#65BF73', textAlign: 'right'}}>
+          {Number(data.totalTime) + Number(data.prepTime)} min
+        </h1>
+      </div>
+
+      {data.mainImage?.fluid == null ? (
+        <p> Image not found</p>
+      ) : (
+        <Img alt="" fluid={data.mainImage?.fluid} />
```

Comment:
For accessibility, maybe we should render the image's `description` from Contentful?